winpr/include/winpr/synch.h

```diff
@@ -293,9 +293,18 @@ extern "C"
 	InitializeCriticalSectionAndSpinCount(lpCriticalSection, dwSpinCount)
 #endif
 
+#if defined(__clang__)
```

Comment:
what about putting this enter block and exit block in a specific header and just include it ?

Reply:
good idea, so we could catch compiler differences better.